neqo-transport/src/stats.rs

```diff
@@ -274,6 +274,9 @@ pub struct Stats {
     /// Whether the connection was resumed successfully.
     pub resumed: bool,
 
+    /// The packet number of the first Initial packet sent.
+    pub first_initial_pn: Option<packet::Number>,
```

Comment:
We shouldn't need to record this.  Maybe if that is your method for testing this, but I'd prefer that the packet be decrypted instead.